packages/gguf/src/types.ts

```diff
@@ -1,3 +1,7 @@
+import type { TransformerLLM } from "./transformer-llm";
+import { LLM_ARCHITECTURES } from "./transformer-llm";
+export * from "./transformer-llm";
```

Comment:
```suggestion
```

this import is not needed

Reply:
Removed with https://github.com/huggingface/huggingface.js/pull/640/commits/09b70cda0654d48005b783bae92369f81a11783c